src/pages/dho/Organization.vue

```diff
@@ -193,7 +193,7 @@ export default {
     recentPayouts: {
       query: require('~/query/payouts/dao-recent-payouts.gql'),
       update: data => {
-        const { count } = data.getDao.payoutAggregate
+        const { count } = data.queryDao[0].treasury[0].redemptionAggregate
```

Comment:
@evgenibir  In case that there is no dao or trasury this would break the app. Use optional chaining and default value to ensure it doesn't break the app.